include/dev/dram/device.h

```diff
@@ -21,23 +21,31 @@
  *
  * mr2      Stores current value of MR2 in DRAM.
  * mr13     Stores current value of MR13 in DRAM.
+ * cal      Pointer to current DRAM freq cal structure.
+ * cfg      Pointer to current DRAM freq cfg structure.
  */
 typedef struct dram_dev_t
 {
-    uint8_t mr2;
-    uint8_t mr13;
+    uint8_t         mr2;
+    uint8_t         mr13;
```

Comment:
What are you thoughts on having mr2 modified in freq_cfg? I think mr13 should be independent, but I'm torn on what to do for mr2. I don't like thinking that we're modifying the value in the table, but also maybe we should be?

Reply:
I think modifying the value in the table is useful for tracking the value.  I have updated the DRAM functions to only use the MR13 value from the table, and no longer stores this in the DRAM device.